components/key-exchange/src/exchange.rs

```diff
@@ -50,6 +69,10 @@ where
     /// * `point_addition_receiver` - The point addition receiver instance used during key exchange
     /// * `executor`                - The MPC executor
     /// * `config`                  - The config used for the key exchange protocol
+    #[cfg_attr(
+        feature = "tracing",
+        tracing::instrument(level = "info", skip(channel, executor), ret)
```

Comment:
wu must also skip `point_addition_sender` and `point_addition_receiver` since the contain points from which session keys can be derived